.prettierignore

```diff
@@ -1,16 +1,19 @@
-out/
+app/node_modules/
```

Comment:
Same here.

Reply:
👍 I _think_ I can undo this change given it was related to eslint. But I think the file needed a tidy-up anyway.